### PR TITLE
Enhancements to make the Docker container easier to use

### PR DESCRIPTION
**Pull request by mraineri**
Added public-rackmount1 as a built-in mockup

### CODE REVIEW

File: redfishMockupServer.py [hunk 1]
```diff
@@ -694,15 +695,21 @@ def main():
         shortForm = args.short_form
         ssdpStart = args.ssdp
 
+        # If inside a Docker container, expose on all interfaces
+        if os.environ.get('INSIDE_DOCKER_CONTAINER', False):
```
Comment (glimchb) [inline redfishMockupServer.py]: instead of using ENV and splitting logic between inside and ouside of docker, why you just always bind to 0.0.0.0 instead?

Comment (mraineri) [reply redfishMockupServer.py]: Fixed; made a change to the Dockerfile to specify 0.0.0.0 to avoid having to do this.

**Review by glimchb — APPROVED** [review]
lgtm

**Comment by mraineri** [comment]
Approved 11/13

**Pull request merged**